### PR TITLE
Media browser and tag editor improvements

### PR DESCRIPTION
**Pull request by Matt95**
See individual commits for details but essentially:
- clean up media browser appearance & usability 
- change to filter by default for media browser search
- move upload to separate component   
- move scene tags to tag editor and enable click to add
- update default layout
- general bug fixes (list mode, button text etc)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -80,10 +82,11 @@
     "react-monaco-editor": "^0.7.3",
     "react-newline-to-break": "^1.0.6",
     "react-rectangle": "1.3.3",
-    "react-resumable-js": "https://github.com/aaronpcz/ReactResumableJS#1.1.27",
+    "react-resumable-js": "git+https://github.com/aaronpcz/ReactResumableJS.git#1.1.27",
```
Comment (Matt95) [inline package.json]: Only way I could get this to work, break anything? @aaronpcz

File: src/js/stores/grid-store.js [hunk 2]
```diff
@@ -97,6 +97,16 @@ var GridStore = assign({}, EventEmitter.prototype, {
     getFocusedMediaObject: function () {
         return gridState.focusedMediaObject;
     },
+
+    getFullFocusedMediaObject: function() {
```
Comment (Matt95) [inline src/js/stores/grid-store.js]: Later removed was a bad idea

File: src/js/components/navigation-bar.jsx [hunk 3]
```diff
@@ -121,7 +121,7 @@ var NavigationBar = React.createClass({
                         <label htmlFor="scene-graph-input" className="mf-text">Graph Viewer</label>
                         <input type='text' ref='name' id="scene-graph-input" className='form-control'
                                placeholder='Room ID'/>
-                        <button type='submit' className='btn btn-dark'>set Room</button>
+                        <button type='submit' className='btn btn-dark'>Set Room</button>
```
Comment (Matt95) [inline src/js/components/navigation-bar.jsx]: JIRA CORE-122 was CORE-114 something else?

Comment (aaronpcz) [reply src/js/components/navigation-bar.jsx]: Yeah looks like I've added a redundant ticket.  I've deleted one of them.

File: package.json (hunk 1 above)
Comment (aaronpcz) [reply package.json]: Should be fine - interesting - must be some difference between npm version used.

Not super important but post your node + npm version.
I think we should specify in the package json the engines, allowing us to share the same node and npm version.

Comment (aaronpcz) [reply package.json]: I bet the later versions of npm require the git+

The earlier versions of npm had a lot to learn from other dependency management tools.  Looks like they are slowly but surely getting there.

**Pull request merged**